test/clt-tests/core/test-freeze-unfreeze-update.rec

```diff
@@ -75,3 +74,31 @@ mysql -P9315 -h0 -e "DESCRIBE t"
 mysql -P9315 -h0 -e "UPDATE t SET s='с' WHERE id=1; FLUSH RAMCHUNK t"; echo $?
```

Comment:
You don't test that `s` becomes `c` in the following commands.

Reply:
I changed the logic for this part of the test as well.